testing/setup/kit-operator.cloudformation.yaml

```diff
@@ -0,0 +1,75 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
It would be sweet if the KIT project vended this to us, so we didn't need to copy it around.

Reply:
I agree. We can consider changing this in the future as the IaC matures.